iclr_wrap_up/experiment_grid.py

```diff
@@ -0,0 +1,178 @@
+import importlib
+import pandas as pd
+import datetime
+import os
+import json
+
+from sacred import Experiment
+from sacred.observers import MongoObserver
+
+from tensorflow.python.keras import backend as K
+
+from tensorflow.python.keras.callbacks import TensorBoard
+from iclr_wrap_up.callbacks.loggingreporter import LoggingReporter
+from iclr_wrap_up.callbacks.metrics_logger import MetricsLogger
+from iclr_wrap_up.callbacks.activityprojector import ActivityProjector
+import matplotlib
+matplotlib.use('agg')
+
+import iclr_wrap_up.credentials as credentials
+if "HYPERPARAMID" in os.environ:
+    h_id = int(os.environ["HYPERPARAMID"])
+    h_file = open("hyperparameter.json")
+    h_data = json.load(h_file)["hyperparameter"][h_id]
+    exp_name = h_data["name"]
+else:
+    line = open('experiment_grid.sge').readlines()[1] #that's the line -N ...
+    exp_name = line[line.find("-N")+3:].strip()
+
+ex = Experiment(exp_name)
+
+url = f'mongodb://{credentials.MONGODB_ADMINUSERNAME}:{credentials.MONGODB_ADMINPASSWORD}@{credentials.MONGODB_HOST}/?authMechanism=SCRAM-SHA-1'
+ex.observers.append(MongoObserver.create(url=url,
+                                         db_name=credentials.MONGODB_DBNAME))
+
+
+@ex.config
+def hyperparameters():
+    h_id = int(os.environ["HYPERPARAMID"])
```

Comment:
can me make the id a string, instead of just the position in the list. I believe that this way of addressing configs is very brittle.